packages/react/src/lavadome.jsx

```diff
@@ -48,15 +56,16 @@ function tokenToText(token) {
     return text;
 }
 
-export const LavaDome = ({ text, unsafeOpenModeShadow }) => {
-    // variable @text is named that way only for visibility - in reality it's a lavadome token
-    const token = text, host = useRef(null);
+export const LavaDome = ({ token, unsafeOpenModeShadow }) => {
+    const host = useRef(null);
 
     return (
         // form a span to act as the LavaDome host
         <span ref={host}>
             <LavaDomeShadow
-                host={host} token={token}
+                host={host}
+                // accept both formats (token, or [token, copy])
```

Comment:
why?

Reply:
Originally, so that users can ignore the `copy` option and just do `const token = toLavaDomeCapabilities(secret);`, but I think to cancel that would mean they'd have to do `const [token] = toLavaDomeCapabilities(secret);` which is fine, so i need to see if that's the only reason which makes this a wrong choice or not

Reply:
would prefer to do `[token]`/`[token, copy]` if there's no great reason just for simplicity sake.  I assume this is a React thing that means you can't just use an object and instead need to deal with arrays

Reply:
c9f2ec4 (you're right about the object part too)